lib/features/composer/presentation/composer_controller.dart

```diff
@@ -800,7 +813,7 @@ class ComposerController extends BaseController {
       showConfirmDialogAction(context,
           AppLocalizations.of(context).message_dialog_send_email_without_recipient,
           AppLocalizations.of(context).add_recipients,
-          onConfirmAction: () => {},
+          onConfirmAction: () => {isSendEmailLoading.value = false},
```

Comment:
Please set `isSendEmailLoading = false` in `onCancelAction`

Reply:
https://github.com/linagora/tmail-flutter/assets/80142234/31e41933-2e19-4b1b-b9fe-803ae53cb48b

There are some cases clicking on confirm button after `isSendEmailLoading` is true so i thought set `isSendEmailLoading = false` in `onConfirmAction` is better.